app/src/main/java/org/fossasia/openevent/app/common/app/binding/BindingAdapters.java

```diff
@@ -107,17 +112,44 @@ private static void setGlideImage(ImageView imageView, String url, Drawable draw
             .centerCrop()
             .transition(withCrossFade())
             .transform(transformation == null ? new CenterCrop() : transformation)
+            .listener(glidePalette)
             .into(imageView);
     }
 
     @BindingAdapter(value = {"imageUrl", "placeholder"}, requireAll = false)
     public static void bindDefaultImage(ImageView imageView, String url, Drawable drawable) {
-        setGlideImage(imageView, url, drawable, null);
+        setGlideImage(imageView, url, drawable, null, null);
     }
 
     @BindingAdapter(value = {"circleImageUrl", "placeholder"}, requireAll = false)
     public static void bindCircularImage(ImageView imageView, String url, Drawable drawable) {
-        setGlideImage(imageView, url, drawable, new CircleCrop());
+        setGlideImage(imageView, url, drawable, new CircleCrop(), null);
+    }
+
+    @BindingAdapter(value = {"eventImageUrl", "placeholder"}, requireAll = false)
+    public static void bindNavHeader(View navHeader, String url, Drawable drawable) {
```

Comment:
but this is better option than adding this code into activity. Any other way to do this?